com.woltlab.wcf/templates/trophyList.tpl

```diff
@@ -3,7 +3,7 @@
 {capture assign='contentHeader'}
 	<header class="contentHeader messageGroupContentHeader">
 		<div class="contentHeaderTitle">
-			<h1 class="contentTitle">{$category->getTitle()}</h1>
+			<h1 class="contentTitle">{$category->getTitle()} <span class="badge badgeInverse">{#$items}</span></h1>
```

Comment:
Strange, it looks okay when reviewing the file in your repository: https://github.com/jens1o/WCF/commit/3f08bfd8b26122d3f322808eb5a39132263d3531#diff-0f077676cc079dc0e9ea7493ec05a6d3